Unable to build samples/net/wifi/shell with NXP M.2 Wi-Fi and BT Shield

<!--
**Notes**
Github Discussions (https://github.com/zephyrproject-rtos/zephyr/discussions)
are available to first verify that the issue is a genuine Zephyr bug and not a
consequence of Zephyr services misuse.

This issue list is only for bugs in the main Zephyr code base
(https://github.com/zephyrproject-rtos/zephyr/). If the bug is for a project
fork (such as NCS) specific feature, please open an issue in the fork project
instead.
-->

**Describe the bug**
Following the instructions from https://docs.zephyrproject.org/latest/boards/shields/nxp_m2_wifi_bt/doc/index.html for building the Wi-Fi sample results in linker errors
<!--
A clear and concise description of what the bug is.

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 - What have you tried to diagnose or workaround this issue?
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
 - ...
-->

**To Reproduce**
```shell
$ west build -b mimxrt1060_evk@C//qspi --shield nxp_m2_2el_wifi_bt samples/net/wifi/shell
```
<!--
Steps to reproduce the behavior:
1. mkdir build; cd build
2. cmake -DBOARD=board\_xyz
3. make
4. See error
-->

**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->
The build succeeds

**Impact**
<!--
What impact does this issue have on your progress (e.g., annoyance, showstopper)
-->
Annoyance

**Logs and console output**
<!--
If applicable, add console logs or other types of debug information
e.g Wireshark capture or Logic analyzer capture (upload in zip archive).
copy-and-paste text and put a code fence (\`\`\`) before and after, to help
explain the issue. (if unable to obtain text log, add a screenshot)
-->
```
/home/pdgendt/.local/opt/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/wifi/libdrivers__wifi.a(nxp_wifi_drv.c.obj): in function `k_event_clear':
/home/pdgendt/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/syscalls/kernel.h:862: undefined reference to `z_impl_k_event_clear'
/home/pdgendt/.local/opt/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/wifi/libdrivers__wifi.a(nxp_wifi_drv.c.obj): in function `k_event_init':
/home/pdgendt/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/syscalls/kernel.h:793: undefined reference to `z_impl_k_event_init'
/home/pdgendt/.local/opt/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/wifi/libdrivers__wifi.a(nxp_wifi_drv.c.obj): in function `k_event_clear':
/home/pdgendt/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/syscalls/kernel.h:862: undefined reference to `z_impl_k_event_clear'
/home/pdgendt/.local/opt/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/wifi/libdrivers__wifi.a(nxp_wifi_drv.c.obj): in function `k_event_wait':
/home/pdgendt/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/syscalls/kernel.h:881: undefined reference to `z_impl_k_event_wait'
/home/pdgendt/.local/opt/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/drivers/wifi/libdrivers__wifi.a(nxp_wifi_drv.c.obj): in function `k_event_clear':
/home/pdgendt/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/syscalls/kernel.h:862: undefined reference to `z_impl_k_event_clear'
```

**Environment (please complete the following information):**

 - OS: Ubuntu 24.04
 - Toolchain: zephyr SDK 0.16.8
 - Commit SHA or Version used: 1596cc5667b

**Additional context**
<!--
Add any other context that could be relevant to your issue, such as pin setting,
target configuration, ...
-->

--- a/drivers/wifi/nxp/Kconfig.nxp
+++ b/drivers/wifi/nxp/Kconfig.nxp
@@ -10,6 +10,7 @@ menuconfig WIFI_NXP
 	select SDHC if !SOC_SERIES_RW6XX
 	select SDIO_STACK if !SOC_SERIES_RW6XX
 	select WIFI_NM
+	select EVENTS
 	depends on DT_HAS_NXP_WIFI_ENABLED
 	help
 	  Enable NXP SoC Wi-Fi support.

